modules/ROOT/pages/about-designing-a-mule-application.adoc

```diff
@@ -4,10 +4,52 @@ include::_attributes.adoc[]
+
+To start creating Mule applications:
+
+* Try the easiest option, which is to import a template into Flow Designer from Anypoint Exchange. See xref:import-templat.adoc[Create a Mule Application Quickly from a Template].
+* Create a new Mule-app project in Design Center and use the wizard that automatically opens to walk you through scaffolding a flow with a trigger card and a target card.
+* Create a new Mule-app project and click *Go straight to the canvas* in the wizard to start building a flow from a blank canvas.
+
+== Tour of the Flow Designer Interface
+
+.The five areas of the Flow Designer interface
+image::FD-annotated.png[]
+
+. The canvas
++
+Here you can create a flow. The cards in a flow appear in the middle, in a sequence from left to right. The first card in a flow is the trigger card, which represents the connector and operation that you want to use to trigger the flow.
++
+In the top-left corner of the canvas are the names of the project and the flow.
+. The panes
++
+Click the top icon to open the *Project* pane, which lists a project's flows, the configurations that are being used, and any global data types that have been defined.
++
+Click the bottom icon to open the *Dependencies* pane, which lists a project's dependencies.
++
+What are reusable configurations?:: When you add a card that connects to another system, you specify how Flow Designer can locate and authenticate to that system. For example, if you add a Database card to a flow, you need to specify the type of database, its host, other connectivity information, and how to authenticate. The set of this information is called a _configuration_. After you add one Database card, the next Database card you add uses the same configuration by default, though you can change it. In the Project pane, the list of resuable configurations is the list of all of the configurations that you have created in a project and that can be used in additional cards that connect to the same systems as cards that are already in the project.
++
+What are data types?:: Data types show the formats in which data are used in a flow. A local data type is defined within a particular card. A global data type is defined for a project and can be reused in multiple cards. See xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer] for examples of how you can create and use global data types.
++
+What are dependencies?:: A dependency is any connector, core Mule Runtime component, API, or module that you use in a flow. The first time that you add one of those objects to a flow, you add the current version of it. Every time afterward that you use that object again in the flow, you use the same version of it. From time to time, that object might be updated in Exchange, the update fixes bugs or introducing enhancements. If you would like your flow to benefit from the update, you can update the version of the object that you are using in your flow.
```

Comment:
I suggest removing "of it." It is not needed. The sentence that starts with "From time to time" is not grammatical. Maybe change to ", with the update fixing bugs or..."

Reply:
I'll keep "of it". I think it reads better.

As for the sentence that starts with "From time to time", I've removed ", the update fixing bugs or introducing enhancements".